src/Uno.UI/UI/Xaml/Controls/AppBar/AppBar.cs

```diff
@@ -156,16 +156,7 @@ protected override Size MeasureOverride(Size availableSize)
 			// (It's the responsibility of each child to constraint itself)
 			// Note: This override is used only for the XAML command bar, not the native!
```

Comment:
So is the comment wrong here, and this method does affect the native command bar rendering somehow? What's the link with the related issue, which involves the native command bar?

Reply:
This is marked as resolved but I'd still like to know the answer :)